plugins/inquiry/app/mailers/inquiry/inquiry_mailer.rb

```diff
@@ -81,7 +82,17 @@ def notification_new_project(inform_dl, inquiry, user_full_name)
           subject += "/#{@inquiry.tags["domain_name"]}"
         end
       end
-      mail(to: inform_dl, subject: subject, content_type: "text/html")
+      
+      # Render the email body content
+      email_body = render_to_string('inquiry/inquiry_mailer/notification_new_project', layout: false)
+
+      send_custom_email(
+        recipient: inform_dl,
+        subject: subject,
+        body_html: email_body
+      )
+      # mail(to: inform_dl, subject: subject, content_type: "text/html")
```

Comment:
Please remove this comment

Reply:
Yes yes! I just leave it so you can see the difference